src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -5626,6 +5626,28 @@ def test_aks_create_disable_rbac(self, resource_group, resource_group_location):
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
```

Comment:
Queued another [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=55325098&view=results) to check these cases. From test result, the automatically created workspace id cannot be mocked, please mark these cases as `@live_only()`.